src/KvnrValidator.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+/**
+ * @author    Davide Danna
+ * @copyright 2020 GAIA AG, Hamburg
+ * @package   KVNR-Validator
+ *
+ * Created using PhpStorm at 07.09.20 10:36
+ */
+
+namespace GaiaGroup;
+
+/**
+ * Class for KVNR code validation
+ */
+class KvnrValidator
+{
+    /**
+     * Checks if given KVNR code is correct by validating its format and by verifying
+     * its checksum (ref: https://de.wikipedia.org/wiki/Krankenversichertennummer)
+     *
+     * @param string $kvnr KVNR string to be validated
+     *
+     * @return bool true / false
+     */
+    public static function validate(string $kvnr): bool
+    {
+        $isValid = false;
+
+        // checks KVNR format using regex
+        if (preg_match("/^[A-Z][0-9]{9}$/", $kvnr)) {
+            // computes CRC digit using KVNR string trimmed out of the last digit
+            $kvnrCrc = self::computesKvnrCrcDigit(substr($kvnr, 0, strlen($kvnr) - 1));
+
+            // compares computed CRC values with last digit in provided KVNR
+            $isValid = $kvnr[9] == $kvnrCrc;
+        }
+
+        return $isValid;
```

Comment:
```suggestion
        // checks KVNR format using regex
        if (preg_match("/^[A-Z][0-9]{9}$/", $kvnr)) {
            // computes CRC digit using KVNR string trimmed out of the last digit
            $kvnrCrc = self::computesKvnrCrcDigit(substr($kvnr, 0, strlen($kvnr) - 1));

            // compares computed CRC values with last digit in provided KVNR
            return $kvnr[9] == $kvnrCrc;
        }
        return false;
```
I think this is easier to read